content/klustered/klustered-klustered-teams-jetstack-crashbeerbackoff.hcl

```diff
@@ -0,0 +1,56 @@
+episode "klustered-klustered-teams-jetstack-crashbeerbackoff" {
+    title =  "Klustered Teams: Jetstack & CrashBeerBackOff"
+    draft = false 
+    show = "klustered"
+    scheduled_for = "2022-08-11"
```

Comment:
Just need this date fixed and I can merge

Reply:
🤔 how's it looking now?